src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1147,6 +1147,17 @@ void goto_instrument_parse_optionst::instrument_goto_program()
     goto_model.goto_functions.update();
   }
 
+  if(cmdline.isset("synthesize-loop-invariants"))
+  {
+    log.warning() << "Loop invariant synthesizer is still work in progress. "
```

Comment:
I am not sure this is needed as it will be removed by your next PR.

Reply:
Before the next PR is merged, users may see and want to use this flag. This warning notifies them this flag is just a placeholder now.

Reply:
OK.